src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -47,9 +47,17 @@ async function boot(userOptions?: Partial<BlazorOptions>): Promise<void> {
     return true;
   };
 
+  window.addEventListener('unload', sendDisconnect, false);
+
   window['Blazor'].reconnect = reconnect;
 
   logger.log(LogLevel.Information, 'Blazor server-side application started.');
+
+  function sendDisconnect(_: Event): void {
```

Comment:
The `_` param is strange. Does the compiler complain if you just make it `sendDisconnect()`? TBH it might be nicer to inline this into line 50 anyway.